### PR TITLE
`dimensions` section in project file.

### PR DESCRIPTION
**Pull request by rec**
Another thing to make projects easier.

You can now set a "dimensions" section in the project file or the `--dims` flag command line and the program works out the right `num`, `layout`, and parameters to the layout (`width` and `height` or `x`, `y`, `z`.

This means that nearly all the time you don't need to specify a layout section at all... :-)

### CODE REVIEW

File: bibliopixel/project/fix.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import copy
```
Comment (adammhaile) [inline bibliopixel/project/fix.py]: Why is this called fix?

Comment (rec) [reply bibliopixel/project/fix.py]: Uh, good question.  :-)

The idea is that they "fix" up a project description.

Not a great name izzit?  Ideas?

**Comment by adammhaile** [comment]
Nope... I hate it. It's ambiguous. Even cleanup would be better.

**Comment by rec** [comment]
`cleanup` is better but there must be something better than that.

Let me think about this a bit more, then... :-)

**Comment by rec** [comment]
I decided `cleanup` was as good as anything and I made the change. What do you think?

**Comment by adammhaile** [comment]
Works for me.

**Pull request merged**